tests/OSPSuite.Core.Tests/Core/DataRepositoryTaskSpecs.cs

```diff
@@ -86,6 +86,49 @@ public void should_have_set_the_name_of_the_data_repository_in_the_table()
       }
    }
 
+   public class When_converting_a_data_repository_with_auxiliary_columns_to_datatable : concern_for_DataRepositoryTask
+   {
+      private DataColumn _col1;
+      private DataColumn _col2;
+      private DataTable _dataTable;
+      private IEnumerable<DataTable> _results;
+
+      protected override void Context()
+      {
+         base.Context();
+         _col1 = new DataColumn("col1", _mass, _baseGrid1)
+         {
+            Values = new float[] {10, 20, 30},
+            DataInfo = {Origin = ColumnOrigins.Observation}
+         };
+         _col2 = new DataColumn("col2", _mass, _baseGrid1)
+         {
+            Values = new float[] {100, 200, 300},
+            DataInfo = {AuxiliaryType = AuxiliaryType.ArithmeticStdDev, Origin = ColumnOrigins.ObservationAuxiliary}
+         };
+
+         _col1.AddRelatedColumn(_col2);
+         _dataRepository.Add(_col2);
```

Comment:
Here: I had Error first and then conc. Before the export would have been as is. Now it follows the convention